tests/test_fields.py

```diff
@@ -1165,7 +1170,7 @@ class TestDateTimeField(FieldValues):
         datetime.date(2001, 1, 1): ['Expected a datetime but got a date.'],
     }
     outputs = {
-        datetime.datetime(2001, 1, 1, 13, 00): '2001-01-01T13:00:00',
+        datetime.datetime(2001, 1, 1, 13, 00): '2001-01-01T13:00:00Z',
```

Comment:
Side note - changed the datetime output from unaware to aware/UTC datetime, since the field itself is datetime-aware.